### DISCUSSION TITLE
Docker compose?

### DISCUSSION THREAD

**Discussion by JacobDK8200** [xbapps/xbvr#900] [General]
Hi

Bit of a linux noob here. 

Im using portainer to load docker compose files.  They are running on a open media vault(debian) based Arm board.

How would i translate from docker run commands to docker compose? 

I want the xbvr config on the harddrive (not the system sd card) f.eks.:  /srv/myharddrive/APP_data/xbvr :/config How do I do that?

Can I change the port XBVR is using?

Thanks alot!

**Comment by uslacker99** [comment]
not an expert here, but here's what mine looks like. I used stashdb as a template.

`# APPNICENAME=XBVR
# APPDESCRIPTION=An organizer for your porn, written in Go
version: '3.4'
services:
  xbvr:
    image: ghcr.io/xbapps/xbvr:latest
    container_name: xbvr
    privileged: true
    restart: unless-stopped
    ## the container's port must be the same with the STASH_PORT in the environment section
    ports:
      - "9999:9999"
    ## If you intend to use stash's DLNA functionality uncomment the below network mode and comment out the above ports section
    # network_mode: host
    logging:
      driver: "json-file"
      options:
        max-file: "10"
        max-size: "2m"
    environment:
      #- STASH_CACHE=/cache/
      ## Adjust below to change default port (9999)
      - STASH_PORT=9999
    volumes:
      - /etc/localtime:/etc/localtime:ro
      ## Adjust below paths (the left part) to your liking.

      ## Keep configs, scrapers, and plugins here.
      #- xbvr-config:/root/.config/
      ## NFS Volumes
      - /home/Megalodon:/home/Megalodon
      - /home/Muscle:/home/Muscle
      - /home/Minime:/home/Minime
      - /home/Colossus:/home/Colossus
`

**Reply by theRealKLH** [reply]
You can use ANY pet you wish when running via Docker...

ports:
- "6666:9999"

First number is the port you want to use...

Http://192.168.1.1:6666
